### PR TITLE
Move the Linux service to the main block

### PR DESCRIPTION
**Pull request by pkgw**
It wants to access Redis, etc., so it should all go together.

Also comment out the "datatier" storage, which isn't used in production. (There's probably a better variable-ier way to do this.)

I wanted to rename the current "wwt" service/plan to "communities" to reflect its reduced role, but Terraform would want to destroy and recreate the resources in order to do so, which I don't want to do even though the whole point of Terraform is that it's supposed to be safe ...

### CODE REVIEW

File: website/.terraform.lock.hcl [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# This file is maintained automatically by "terraform init".
```
Comment (coderandhiker) [inline website/.terraform.lock.hcl:1]: @pkgw this should be excluded from your commit - add a record to your .gitignore for everything under .terraform and the .terraform.lock.hcl if the aren't in there already

**Review by coderandhiker — CHANGES_REQUESTED** [review]
We cannot dual deploy Windows and Linux App Service Plans into the same Resource Group at this time.  This is on the roadmap: https://feedback.azure.com/forums/169385-web-apps/suggestions/37287583-allow-a-linux-and-windows-app-service-plan-to-exis

I recommend adding a new var.prefix-linux-resources resource group and updating the app service plan and app service to deploy into it.

File: website/.terraform.lock.hcl (hunk 1 above)
Comment (pkgw) [reply website/.terraform.lock.hcl:1]: @coderandhiker The Terraform CLI specifically instructs that these lockfiles should be kept in version control. I think it's a more recent thing, it only appeared when I upgraded my OS a week or two ago.

Comment (coderandhiker) [reply website/.terraform.lock.hcl:1]: Ok, that's new to me - go ahead and add it back in. I found the ref: https://discuss.hashicorp.com/t/terraform-0-14-the-dependency-lock-file/15696

**Review by coderandhiker — APPROVED** [review]
I ran the revisions in my subscription and everything looks good now.

**Pull request merged**